README.md

```diff
@@ -289,6 +289,14 @@ And an answers, additional, or authority looks like this
 }
 ```
 
+#### `RP`
+
+``` js
+{
+  data: mbox-dname
```

Comment:
Need to update this too.

Reply:
thank you for guiding me through this, @silverwind - much appreciated!